File: app/src/main/java/com/gopher/guardian/view/nextofkin/NextOfKinFragment.java
package com.gopher.guardian.view.nextofkin;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.fragment.app.Fragment;
import com.gopher.guardian.R;

public class NextOfKinFragment extends Fragment {

  public NextOfKinFragment() {
    // Required empty public constructor
  }

  @Override
  public void onCreate(final Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
  }

  @Override
  public View onCreateView(
      final LayoutInflater inflater, final ViewGroup container, final Bundle savedInstanceState) {
    return inflater.inflate(R.layout.fragment_next_of_kin, container, false);
  }
}

File: app/src/main/java/deakin/gopher/guardian/view/nextofkin/NextOfKinFragment.kt
package deakin.gopher.guardian.view.nextofkin

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import deakin.gopher.guardian.R

class NextOfKinFragment : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_next_of_kin, container, false)
    }
}
